src/command/process_update_task.rs

```diff
@@ -344,6 +508,68 @@ impl UpdateProcessor {
     }
 }
 
+#[derive(Serialize, Deserialize, Debug, TypedBuilder, Eq, PartialEq, Clone)]
+#[serde(crate = "fang::serde")]
+pub struct ScheduleWeatherTask {
```

Comment:
I think it can mobed into a separate module, because it's not related to processing user commands

Reply:
I think is fine in that module because it is scheduled as a part of an Update so it is kind a part of a processing update.

Reply:
Moved to deliver module.